src/Components/BookShelfModal/ModalContent.js

```diff
@@ -0,0 +1,19 @@
+import { BookCardSixColum } from "../BookCard/BookCard";
+
+export function ModalContent({ bookShelfCase }) {
+  const bookCard =
+    bookShelfCase.bookShelfCase &&
+    bookShelfCase.bookShelfCase.books.map(
+      ({ id, bookCoverImg, bookName, writer }) => (
+        <BookCardSixColum
+          id={id}
+          key={id}
+          bookCoverImg={bookCoverImg}
+          bookName={bookName}
+          writer={writer}
+        />
+      )
+    );
+
+  return <>{bookCard}</>;
```

Comment:
```suggestion
export function ModalContent({ bookShelfCase }) {
  return <>{
    bookShelfCase.bookShelfCase?.books.map(
      ({ id, bookCoverImg, bookName, writer }) => (
        <BookCardSixColum
          id={id}
          key={id}
          bookCoverImg={bookCoverImg}
          bookName={bookName}
          writer={writer}
        />
      )
    }</>;
```
jsx 요소는 그냥 return 안쪽에 바로 작성하셔도 될 것 같습니다

Reply:
✔ 수정했습니다!